app/src/main/java/campus/tech/kakao/contacts/ContactsAdapter.kt

```diff
@@ -0,0 +1,68 @@
+package campus.tech.kakao.contacts
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+
+class ContactsAdapter(private var contList: List<Contact>, private val listener: OnItemClickListener, private val longListener: OnItemLongClickListener): RecyclerView.Adapter<ContactsAdapter.ViewHolder>() {
```

Comment:
클릭이벤트 처리를 위해 interface 정의와 구현이 포함되면서 Adapter 함수가 하는 일에 비해 조금 길어진 느낌이 있네요.

코틀린 고차함수를 적절히 활용하면, OnItemClickListener, OnItemLongClickListener interface를 쓰지 않고, ViewHolder가 View.OnClickListener와 View.OnLongClickListener를 구현하지 않아도 더 간단하게 클릭이벤트를 처리할 수 있습니다.

```suggestion
class ContactsAdapter(
    private var contList: List<Contact>, 
    private val onClick: (Int) -> Unit, 
    private val onLongClick: (Int) -> Unit
): RecyclerView.Adapter<ContactsAdapter.ViewHolder>() {
```